protect-api/README.md

```diff
@@ -0,0 +1,114 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- java
+page_type: sample
+name: "Java API written in Spring Boot that both protects its own endpoints"
+description: "This Java API that protects it's own endpoints using JWT scope validation. The code in this sample is used by one or more articles on docs.microsoft.com."

```

Comment:
it's should be its

Reply:
Indeed!  Good catch @skabou, addressed in https://github.com/Azure-Samples/ms-identity-docs-code-java/pull/4/commits/79b1d634b85d249428c2151503cde0d17cba095f

Reply:
Marking resolved (@skabou doesn't have permission to do so)